### PR TITLE
Feature/flight zero

### PR DESCRIPTION
**Pull request by jamie-54**
Add Kill and Arm 
Add initial angular velocities 

This is the code currently on the quadcopter

### CODE REVIEW

File: src/radio.cpp [hunk 1]
```diff
@@ -104,7 +104,13 @@ namespace bzzz
 
     bool Radio::kill()
     {
-        return channelData[RADIO_CHANNEL_SWITCH_A] <= 500;
+        return channelData[RADIO_CHANNEL_SWITCH_A] >= 1500;
+    }
+
+    float Radio::readKillSwitch()
```
Comment (alphaville) [inline src/radio.cpp:110]: This is unnecessary. There is `radio.kill()`

**Comment by alphaville** [comment]
I'm closing this because this branch is stale with respect to `dev`. I'll make a new one with the same changes, but focusing on #29 only.

File: src/radio.cpp (hunk 1 above)
Comment (jamie-54) [reply src/radio.cpp:110]: This was added to allow us to read if the kill switch has been turned off again, without it we have to reset the ESP32 every time the kill switch is triggered.

**Comment by jamie-54** [comment]
should we delete some branches that aren't being used then?

**Pull request closed without merging**